openpype/settings/entities/schemas/projects_schema/schema_project_fusion.json

```diff
@@ -74,7 +74,56 @@
                     "type": "dict",
                     "collapsible": true,
                     "key": "CreateSaver",
-                    "label": "Create Saver",
+                    "label": "Create Render Saver",
+                    "is_group": true,
+                    "children": [
+                        {
```

Comment:
@kalisp It seems that the OpenPype settings are lacking the implemented settings for `default_frame_range_option`. Should we still support the backwards compatibility?